boards/arm/lpcxpresso55s36/lpcxpresso55s36.dts

```diff
@@ -17,7 +17,7 @@
 	chosen {
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
-		zephyr,code-partition = &sramx;
+		zephyr,code-partition = &slot0_partition;
```

Comment:
Please fix typo in commit message: s/slot1_aprtition/slot1_partition

Reply:
> Please fix typo in commit message: s/slot1_aprtition/slot1_partition

Fixed. Thank you.